CHANGELOG.md

```diff
@@ -6,51 +6,134 @@ For advice on how to use these release notes see [our guidance on staying up to
 
-If you're not using our page template:
+These changes affect these components:
 
-1. Add the `govuk-template--rebranded` to the `<html>` element of your page to use the rebranded styles of Footer and Cookie banner components.
+- [GOV.UK header](https://design-system.service.gov.uk/components/header/)
+- [GOV.UK footer](https://design-system.service.gov.uk/components/footer/)
+- [Service navigation](https://design-system.service.gov.uk/components/service-navigation/)
+- [Cookie banner](https://design-system.service.gov.uk/components/cookie-banner/)
 
-We've added a new `govukRebrand` ['variable' option to our page template](https://design-system.service.gov.uk/styles/page-template/#changing-template-content). It makes the Header, Footer, Service navigation and Cookie banner components use new styles for the brand refresh and display updated content (new logo in the Header component, crown in the Footer component).
+These changes also affect the `theme-color` metadata and assets, including:
 
-Use [`set`](https://mozilla.github.io/nunjucks/templating.html#set) to assign it `true` and enable the new styles:
+- icons
+- Open Graph image
+- `manifest.json`
+
+To help you get ready, we've published this release, which includes several features to make the necessary updates across your service:
+
+- updated markup for the GOV.UK header component (with the refreshed logo) and GOV.UK footer component (adding the crown logo)
+- a new `govuk-template--rebranded` HTML class to apply updated styles from our CSS
+- a new set of assets for icons, Open Graph image and page metadata
+
+You’ll need to serve the new assets in your service. What you’ll need to do after that depends on the code in your service and if you use GOV.UK Frontend:
+
+- with its Nunjucks page template and macros
+- with its Nunjucks macros (without the page template)
+- without using Nunjucks
+- using the Prototype Kit
+
+##### Add the brand refresh assets to your project
+
+GOV.UK Frontend provides updated assets for the icons, Open Graph image and `manifest.json` to reflect the refreshed brand. These assets are available in the `dist/govuk/assets/rebrand` folder of the package.
+
+If you [serve the assets from the GOV.UK Frontend assets folder](https://frontend.design-system.service.gov.uk/import-font-and-images-assets/#serve-the-assets-from-the-gov-uk-frontend-assets-folder-recommended), make sure the assets inside the `dist/govuk/assets/rebrand` folder are served correctly at `<YOUR-SITE-URL>/assets/rebrand`.
+
+If you [copy the font and image files into your application](https://frontend.design-system.service.gov.uk/import-font-and-images-assets/#copy-the-font-and-image-files-into-your-application), you’ll need to copy the `/node_modules/govuk-frontend/dist/govuk/assets/rebrand` folder to `<YOUR-APP>/assets/rebrand`. If you use an automated task to copy the files, you may need to update your task to automatically copy our new folder.
```

Comment:
Seems a little unusual that this line points to a location in the `node_modules` directory but the prior lines don't. Should maybe be truncated to just the `dist` part

Reply:
Updated to remove the `/node_modules/govuk-frontend` part of the path as discussed on the call.